plugins/network-elements/nuage-vsp/src/com/cloud/agent/api/element/ApplyAclRuleVspCommand.java

```diff
@@ -79,4 +191,47 @@ public boolean executeInSequence() {
         return false;
     }
 
+    @Override
+    public boolean equals(Object o) {
```

Comment:
Implemented <code>equals</code> method for the <code>Command</code> super class, calling it from subclass.